### PR TITLE
[Site Request] Anikiga (#2346)

### LINKED ISSUE #2346: [Site Request] Anikiga
Name of the website:

Anikiga

•	Site:  http://anikiga.com
•	Manga List:  http://anikiga.com/mangalar/
•	Manga example:  http://anikiga.com/manga/sekiro-manga/
•	Chapter online viewer example: http://anikiga.com/manga/sekiro-manga/bolum-2/

Language: Turkish

### PR DESCRIPTION
**Pull request by Robonau**

### CODE REVIEW

**Comment by 27136773** [comment]
May also add an icon for the website (if available) with the same file name as the ID
See: https://github.com/manga-download/hakuneko/tree/master/src/web/img/connectors

File: src/web/mjs/connectors/anikiga.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class Anikiga extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'anikiga';
+        super.label = 'Anikiga';
+        this.tags = [ 'manga', 'turkish' ];
+        this.url = 'http://anikiga.com/';
```
Comment (27136773) [inline src/web/mjs/connectors/anikiga.mjs]: May remove trailing `/`

**Comment by 27136773** [comment]
Image is to large (120 KB), it should be around 1~16 KB with a resolution around 64x64 pixel
Probably needs to be edited...

**Comment by Robonau** [comment]
not sure why that was the case it was a 16x16 icon 
changed to a png from somewhere else on the site

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**